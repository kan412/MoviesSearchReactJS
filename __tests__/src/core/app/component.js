import React from 'react';
import { shallow } from 'enzyme';
import App from '../../../../src/core/app';

describe('<App />', () => {
  it('render without breaking', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
