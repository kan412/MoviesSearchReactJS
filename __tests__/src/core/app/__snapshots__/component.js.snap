// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render without breaking 1`] = `
<Fragment>
  <AppHeaderComponent />
  <ErrorBoundaryComponent>
    <AppMainComponent />
  </ErrorBoundaryComponent>
  <AppFooterComponent />
</Fragment>
`;
